src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -900,7 +900,7 @@ sealed class X : IComparer
 
     public int Compare(object x, object y)
     {
-        return this.x.Compare(x, y);
+        return ((IComparer)this.x).Compare(x, y);
```

Comment:
📝 I'm OK with the approach taken in this pull request to put us back on a more conservative path, but we should file a follow-up issue to consider ways to improve this scenario in the future.